packages/utils/src/index.ts

```diff
@@ -1,3 +1,4 @@
 import { StoryTemplate } from "./internal/StoryTemplate";
+import { initTabListener } from "./tabListener";
```

Comment:
IMO this should be part of core and not util.

Reply:
Moved over now, but we should refactor the entire `core` package sometime soon.